ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/TipsLayout.kt

```diff
@@ -0,0 +1,79 @@
+package org.cru.godtools.tutorial
```

Comment:
for organization, let's move this composable into the `org.cru.godtools.tutorial.layout` package. It might not live there long term, but it will hold it for now.